### DISCUSSION TITLE
Changing final activation function from Relu to Softmax in DenseNet CNN

### DISCUSSION THREAD

**Discussion by LaiaHumbertVidan** [Project-MONAI/MONAI#2911] [Q&A]
Hi, is there a simple way of changing the activation function of the final layers of the DenseNet CNN from Relu to Sigmoid for a binary classification application in Monai?
Many thanks.

**Comment by wyli** [comment]
I think you can use  `model.features` and add your own classifier for the features

https://github.com/Project-MONAI/MONAI/blob/7cfc8ea3c73ab63cfdca2e0cba9302f9286478a7/monai/networks/nets/densenet.py#L247-L250
